module-4/backend/week_two.md

```diff
@@ -1,22 +1,74 @@
 ## Week Two - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
-
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's one difference between ES5 and ES6?
-2. What's the difference between asynchronous and synchronous JavaScript? 
+One difference between ES5 and ES6 is the syntax.
```

Comment:
I love the way you provided an example! Make sure you're able to articulate how the two functions behave differently.